content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Waterfall'
+Description: 'Models a Software Development Life Cycle (SDLC) where the goal is to create a minimum viable product (MVP). Each phase in the SDLC must be completed before moving on to the next phase.'
```

Comment:
```suggestion
Description: 'Models a Software Development Life Cycle (SDLC) where the goal is to create a minimum viable product (MVP).'
```
Extra info doesn't need to be in the description.

Reply:
@SSwiniarski thanks for reviewing! I took out the extra info.